Gemfile.lock

```diff
@@ -5,15 +5,15 @@ PATH
       logstash-core-plugin-api (~> 2.0)
 
 PATH
-  remote: /Users/aaronabraham/code/aaronabraham311/logstash-8.11.0/logstash-core-plugin-api
+  remote: /Users/yash/Documents/logstash-8.11.1/logstash-core-plugin-api
   specs:
     logstash-core-plugin-api (2.1.16-java)
-      logstash-core (= 8.11.0)
+      logstash-core (= 8.11.1)
 
 PATH
-  remote: /Users/aaronabraham/code/aaronabraham311/logstash-8.11.0/logstash-core
+  remote: /Users/yash/Documents/logstash-8.11.1/logstash-core
```

Comment:
anyone's GemFile.lock can be included here, I'm just overriding it with mine so you know the exact environment in which we got this working e2e for debugging purposes next week